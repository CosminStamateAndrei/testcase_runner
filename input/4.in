2 2 2
-9 4 
7 3 
-5 3 
-1 5 
