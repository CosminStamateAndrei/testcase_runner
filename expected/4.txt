Matrix A
  -9    4 
   7    3 
Matrix B
  -5    3 
  -1    5 
Matrix C
  41   -7 
 -38   36 
